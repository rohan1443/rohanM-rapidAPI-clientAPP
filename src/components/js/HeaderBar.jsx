import React from 'react'

const HeaderBar = () => {
  return (
    <div className="header-bar-container">
      <div className="header-content">
        CHEAP TAKEOFFS
      </div>
    </div>
  )
}

export default HeaderBar