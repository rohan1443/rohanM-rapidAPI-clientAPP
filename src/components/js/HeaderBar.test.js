import React from 'react'
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme'

import HeaderBar from './HeaderBar'

describe("Component testing for Header Bar", () => {
  
  test("matching snapshot", () => {
    const wrapper = renderer.create(<HeaderBar />).toJSON()

    expect(wrapper).toMatchSnapshot()
  })

  test("verifying header text has appeared", () => {
  const wrapper = shallow(<HeaderBar />)

  expect(wrapper.find('.header-content').text()).toEqual("CHEAP TAKEOFFS")
  })
})