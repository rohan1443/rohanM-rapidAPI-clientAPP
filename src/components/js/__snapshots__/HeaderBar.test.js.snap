// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component testing for Header Bar matching snapshot 1`] = `
<div
  className="header-bar-container"
>
  <div
    className="header-content"
  >
    CHEAP TAKEOFFS
  </div>
</div>
`;
